client/landing/gutenboarding/onboarding-block/acquire-intent/style.scss

```diff
@@ -142,9 +142,53 @@
 
 .site-title__input-wrapper {
 	display: block;
+	position: relative;
 	min-height: 40px; // make sure there is a tappable area on mobile when the input is empty
 
+	.madlib__input:empty {
+		@include break-small {
+			padding-right: 300px; // there's not enough container width on smaller screens
+		}
+		@include break-medium {
+			padding-right: 400px;
+		}
+	}
+
 	@include break-small {
 		display: inline;
 	}
 }
+
+.site-title__placeholder {
+	display: inline-block;
+	width: 100%;
+	color: var( --studio-gray-5 );
+
+	@include break-small {
+		height: 80px;
+		position: absolute;
+		left: 0;
+		z-index: 2;
+	}
+
+	&::after {
+		content: '';
+		position: absolute;
+		left: 0;
+		right: 0;
+		bottom: -2px;
+		height: 2px;
+		background-image: linear-gradient( to right, rgba( 238, 238, 238, 1 ), rgba( 238, 238, 238, 0.4 ) );
+	}
+}
+
+/* hide the placeholder when input is not empty */
+.madlib__input:not( :empty ) ~ .site-title__placeholder {
+	display: none;
+}
+
+.madlib__input:focus ~ .site-title__placeholder {
+	@include break-small {
+		display: none;
+	}
+}
```

Comment:
It would be great to have a `madlib__input-placeholder` to reuse in site title and vertical input where we have almost [ identical styles ](https://github.com/Automattic/wp-calypso/blob/master/client/landing/gutenboarding/onboarding-block/acquire-intent/vertical-select/style.scss#L81-L125). Not a blocker for this PR but it should be unified soon to maintain consistency when we iterate further on this.